tests/openvino/test_exporters_cli.py

```diff
@@ -90,20 +90,26 @@ class OVCLIExportTestCase(unittest.TestCase):
         ("text-generation-with-past", "opt125m", "int4_asym_g128", 4, 144),
         ("text-generation-with-past", "opt125m", "int4_sym_g64", 4, 144),
         ("text-generation-with-past", "opt125m", "int4_asym_g64", 4, 144),
-        ("text-generation-with-past", "llama_awq", "int4 --ratio 1.0 --sym --group-size 16 --all-layers", 0, 32),
+        (
+            "text-generation-with-past",
+            "llama_awq",
+            "int4 --ratio 1.0 --sym --group-size 8 --all-layers",
+            0,
+            16 if is_transformers_version("<", "4.39.0") else 17,
```

Comment:
to fix the test looks like we need :
```suggestion
            32 if is_transformers_version("<", "4.39.0") else 34,
```